webapp/src/main/webapp/vue-apps/login/components/provider/LoginProvidersMenu.vue

```diff
@@ -29,9 +29,8 @@
           :aria-label="$t('UILoginForm.label.moreProviders')"
           min-width="auto"
           color="primary"
-          class="pa-4"
-          large
           outlined
+          style="background-color:white"
```

Comment:
shouldn't this use some CSS class and take a variable for branding ?

Reply:
no. The branding apply a color for the background of the v-card, but the background for the button must be white : 
<img width="521" height="209" alt="image" src="https://github.com/user-attachments/assets/bec14ae7-7929-4414-b237-d20798b43e98" />

Reply:
Yes, that's normal, the `class="btn"` is the main style of normal buttons of the platform. We shouldn't change to make a specific UX for button for logins.
@srenault-meeds WDYT ?

Reply:
Yes I confirm, we should use class="btn"

Reply:
PR Updated